exploration/number-selection.md

```diff
@@ -92,44 +95,351 @@ ICU MF1 messages using `plural` and `selectordinal` should be representable in M
+  - `true`
+  - `false`
+  - `min2`
+  - `always`
+- `minimumIntegerDigits`
+  - (non-negative integer, default: `1`)
+
+
+> [!NOTE]
+> The following options or option values are being developed during the Technical Preview
+> period.
+
+The following values for the option `style` are _not_ part of the default registry.
+Implementations SHOULD avoid creating options that conflict with these, but
+are encouraged to track development of these options during Tech Preview:
+- `currency`
+- `unit`
+
+The following options are _not_ part of the default registry.
+Implementations SHOULD avoid creating options that conflict with these, but
+are encouraged to track development of these options during Tech Preview:
+- `currency`
+   - valid [Unicode Currency Identifier](https://cldr-smoke.unicode.org/spec/main/ldml/tr35.html#UnicodeCurrencyIdentifier)
+     (no default)
+- `currencyDisplay`
+   - `symbol` (default)
+   - `narrowSymbol`
+   - `code`
+   - `name`
+- `currencySign`
+  - `accounting`
+  - `standard` (default)
+- `unit`
+   - (anything not empty)
+- `unitDisplay`
+   - `long`
+   - `short` (default)
+   - `narrow`
+
+### Default Value of `select` Option
+
+The value `plural` is default for the option `select` 
+because it is the most common use case for numeric selection.
+It can be used for exact value matches but also allows for the grammatical needs of other 
+languages using CLDR's plural rules.
+This might not be noticeable in the source language (particularly English), 
 but can cause problems in target locales that the original developer is not considering.
 
 > For example, a naive developer might use a special message for the value `1` without
 > considering other locale's need for a `one` plural:
 >
 > ```
 > .match {$var}
-> [1] {{You have one last chance}}
-> [one] {{You have {$var} chance remaining}} // needed by languages such as Polish or Russian
-> [*] {{You have {$var} chances remaining}}
+> 1   {{You have one last chance}}
+> one {{You have {$var} chance remaining}} // needed by languages such as Polish or Russian
+>                                          // such locales typically require other keywords
+>                                          // such as two, few, many, and so forth
+> *   {{You have {$var} chances remaining}}
 > ```
 
-Additional options such as `minimumFractionDigits` and others already supported by `:number`
-should also be supported.
 
-If PR [#532](https://github.com/unicode-org/message-format-wg/pull/532) is accepted,
-also add the following `<alias>` definitions to `<function name="number">`:
+### Percent Style
+
+When implementing `style=percent`, the numeric value of the operand
+MUST be divided by 100 for the purposes of formatting.
+
+### Selection
+
+When implementing [`MatchSelectorKeys`](spec/formatting.md#resolve-preferences), 
+numeric selectors perform as described below.
+
+- Let `return_value` be a new empty list of strings.
+- Let `operand` be the resolved value of the _operand_.
```

Comment:
```suggestion
- Let `operand` be the decimal value of _operand_ after formatting options from `:number` have been applied conforming to [the `sampleValue` grammar of UTS 35](https://unicode.org/reports/tr35/tr35-numbers.html#Plural_rules_syntax).
```

The intermediate formatted value may have notation information associated with it, such as "1.0c3" for "1.0K" or potentially "2.3e4" for "2.3*10^4". We should be clear that this is the amount of information we want from the intermediate value, and it also allows us to select on compact values, which may be important in French, for example.

Reply:
"resolved value" has special meaning in our specification (even though it is not highlighted here). It's important to say that `operand` is the "resolved value". We can add requirements *after* that, such as what you suggest.

Note that `operand` is untyped by us. It is either an object or it is a literal.

We say what to do with the literal (which ones are supported and how to parse them)

We mustn't define what the object is. That's up to the implementation. The implementer is responsible for resolving the decimal value, if that's what they need.

Reply:
Hmm, are you referencing this language?

>  The form of the resolved value is implementation defined and the
>  value might not be evaluated or formatted yet.
>  However, it needs to be "formattable", i.e. it contains everything required
>  by the eventual formatting.

So the "resolved value" may or may not be the intermediate value?